timeless/roles/views.py

```diff
@@ -46,7 +52,12 @@ def edit(id):
     )
 
 
-@bp.route("/delete", methods=["POST"])
-def delete():
-    flash("Delete not yet implemented")
+@bp.route("/delete/<int:id>", methods=["POST"])
+def delete(id):
+    roles = Role.query.get(id)
+    if not roles:
+        return abort(HTTPStatus.NOT_FOUND)
+    roles = Role.query.get(id)
+    DB.session.delete(roles)
+    DB.session.commit()
```

Comment:
@ekondrashev I don't like having the ``DB`` in ``views.py``. See how ``CrudAPIView`` is used in other places, that one was implemented to decouple DB from the views, as far as I understood :)

Reply:
@amihaiemil added a puzzle to address this

Reply:
@ekondrashev `CrudAPIView`  shouldn't be used for `Role`, as it is not meant to be returned with a REST api, Better refer to timeless/companies/views.py file to see how it should be implemented. Please change both puzzles accordingly